### PR TITLE
Kevin/feat: Market-Synth-multiplier add crash/boom, step index

### PR DESCRIPTION
**Pull request by kevinw-binary**
# Description

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/markets/_hero.js [hunk 1]
```diff
@@ -8,6 +8,19 @@ import { Container } from 'components/containers'
 import { useLazyVideo } from 'components/hooks/lazy-video'
 import device from 'themes/device'
 
+const MarketHeader = styled(Header)`
+    font-size: 64px;
+    @media ${device.tabletL} {
+        font-size: 40px !important;
```
Comment (suthesan-deriv) [inline src/pages/markets/_hero.js:14]: Is it a must to add the important tag here?

File: src/pages/markets/_hero.js [hunk 2]
```diff
@@ -8,6 +8,19 @@ import { Container } from 'components/containers'
...
+    }
+`
+
+const MarketSubHeader = styled(Header)`
+    font-size: 24px;
+    @media ${device.tabletL} {
+        font-size: 18px !important;
```
Comment (suthesan-deriv) [inline src/pages/markets/_hero.js:21]: same here as well?

**Review by suthesan-deriv — APPROVED** [review]

File: src/pages/markets/_hero.js (hunk 1 above)
Comment (kevinw-binary) [reply src/pages/markets/_hero.js]: fixed

File: src/pages/markets/_hero.js (hunk 2 above)
Comment (kevinw-binary) [reply src/pages/markets/_hero.js]: fixed

**Pull request merged**